src/main/java/net/snowflake/client/jdbc/SnowflakeUtil.java

```diff
@@ -706,6 +708,16 @@ public static void systemSetEnv(String key, String value) {
       field.setAccessible(true);
       Map<String, String> writableEnv = (Map<String, String>) field.get(env);
       writableEnv.put(key, value);
+
+      if (Constants.getOS() == Constants.OS.WINDOWS) {
```

Comment:
please add some clarification in comment why you need to do this

Reply:
Ok, I will added

Reply:
Done, @sfc-gh-dprzybysz